bears/js/HappinessLintBear.py

```diff
@@ -20,5 +20,14 @@ class HappinessLintBear:
     CAN_DETECT = {'Syntax'}
 
     @staticmethod
-    def create_arguments(filename, file, config_file):
+    def create_arguments(filename, file, config_file,
+                         use_spaces: bool=False):
+
```

Comment:
optional:
I think this line doesn't have to be there, up to you to remove it ;)
(But I would advise to do so)